Koblicica Andrei Radu/ProblemeL5/3.10L5/main.c

```diff
@@ -0,0 +1,53 @@
+#include <stdio.h>
+#include <stdlib.h>
+int NrOfBanknotes(int ron)
+{
+    int counter=0;
+    while(ron)
+    {
+        if(ron>=500)
```

Comment:
I would rather work with an array which contains all the possible values of the banknotes - could you use that in order to simplify your code?